plugins/mimecast/komand_mimecast/tasks/monitor_siem_logs/task.py

```diff
@@ -114,6 +136,24 @@ def _filter_and_sort_recent_events(
             ),
             key=itemgetter(EventLogs.FILTER_DATETIME),
         )
+
+        if len(task_output) >= MAX_EVENTS_PER_RUN:
```

Comment:
should this just be `>`, if it's 7500 we can still grab it all?

Reply:
yeah, the other processing path should be quicker so better to use it if possible